poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -273,21 +276,32 @@ public static OPCPackage open(String path, PackageAccess access)
     *             If the specified file doesn't exist, and a parsing error
     *             occur.
     */
-   public static OPCPackage open(File file, PackageAccess access)
-         throws InvalidFormatException {
-       if (file == null) {
-           throw new IllegalArgumentException("'file' must be given");
-       }
-       if (file.exists() && file.isDirectory()) {
+   public static OPCPackage open(File file, PackageAccess access) throws InvalidFormatException {
+       return open(file.toPath(), access);
+   }
+
+    /**
+     * Open a package.
+     * @param path
+     *            The path of the file to open.
+     * @param access
+     *            Type of access to the file that is allowed
+     * @return A PackageBase object, else <b>null</b>.
+     * @throws InvalidFormatException
+     *             If the specified file doesn't exist, and a parsing error
+     *             occur.
+     */
+   public static OPCPackage open(Path path, PackageAccess access) throws InvalidFormatException {
+       if (Files.exists(path) && Files.isDirectory(path)) {
            throw new IllegalArgumentException("file must not be a directory");
        }
 
-       OPCPackage pack = new ZipPackage(file, access); //NOSONAR
+       OPCPackage pack = new ZipPackage(path, access);
```

Comment:
got rid of the //NOSONAR comment here, any reason?

Reply:
no reason it was removed by mistake. to be honest i dont know what this indicates though, so i might just get rid of all of these in the file. i did a search for `nosonar` and `sonar`, and frankly i dont see the value of this comment at all.